components/dashboard/src/components/WorkspaceClassesOptions.tsx

```diff
@@ -15,14 +15,23 @@ import { AllowedWorkspaceClass, DEFAULT_WS_CLASS } from "../data/workspaces/work
 import { MiddleDot } from "./typography/MiddleDot";
 import { useToast } from "./toasts/Toasts";
 import Modal, { ModalBaseFooter, ModalBody, ModalHeader } from "./Modal";
+import { LoadingState } from "@podkit/loading/LoadingState";
 
 interface WorkspaceClassesOptionsProps {
     classes: AllowedWorkspaceClass[];
     defaultClass?: string;
     className?: string;
+    emptyState?: React.ReactNode;
```

Comment:
@mustard-mh could you help me understand when the empty state happens?

Reply:
@mustard-mh maybe that should raise some conflict? I think it's important that we ensure one can always start a workspace.

Reply:
@filiptronicek Yeah, I raised this in team, will find thread later

The result of discussion is to make it simple, we only show error to users on new workspace page so that admin could update settings

Reply:
[Internal chat](https://gitpod.slack.com/archives/C05H5UQBW6Q/p1708623690416279?thread_ts=1708610406.474879&channel=C05H5UQBW6Q&message_ts=1708623690.416279)